root/etc/e-smith/events/actions/nethserver-dc-user-modify

```diff
@@ -40,3 +46,35 @@ if ($? != 0) {
     die("[ERROR] User $userName modification failed\n");
 }
 
+
+#Create a temporary ldif file (root:root 0600)
+my $fileName = File::Temp->new(
+    UNLINK   => 0,
+    TEMPLATE => "$userName.XXXXXXXXXX",
+    DIR => '/var/lib/machines/nsdc/var/spool/'
+);
+
+#create the ldif content
+my $contentFile = qq (
+dn: cn=$userName,cn=Users,$baseDN
```

Comment:
I'm not sure `$userName` is equal to `cn` attribute. This surely works with users created by NethServer, however could not be true for other tools (RSAT, LDAP account manager...). AFAIK the `$userName` corresponds to `sAMAccountName`.

To get DN from sAMAccountName:
```shell
    systemd-run -M nsdc -q -t \
    /usr/bin/ldbsearch -H /var/lib/samba/private/sam.ldb "samaccountname=${userName}" dn | \
    sed -n '/^dn: / { s/\r// ; p ; q }'
```